### PR TITLE
add field for choosing career page image

### PR DESCRIPTION
**Pull request by mikkelklokkerud**

### CODE REVIEW

File: packages/cms/schemas/careerPage.js [hunk 1]
```diff
@@ -20,6 +20,14 @@ export default {
         maxLength: 160,
       },
     },
+    {
+      name: 'mainImage',
```
Comment (klyngen) [inline packages/cms/schemas/careerPage.js:24]: Should this be required? Does not look like the javascript-code expects that this value can be `null` or `undefined`

Comment (mikkelklokkerud) [reply packages/cms/schemas/careerPage.js:24]: made it required 👍

**Comment by klyngen** [comment]
💻 https://fix-career-842.review.test-alv.no

**Review by klyngen — APPROVED** [review]
Looks good :+1:

**Pull request merged**